PostHog/Replay/PostHogReplayIntegration.swift

```diff
@@ -235,6 +252,21 @@
             // manually masked views through view modifier `PostHogMaskViewModifier`
             if view.phIsManuallyMasked {
                 maskableWidgets.append(view.toAbsoluteRect(parent))
+                return
```

Comment:
@ioannisj I wonder if you missed that or if there was a reason to not return early since the view is maskLookingking at the children would not make sense if all childwerel be masked already

Reply:
I renamed `parent` to `window` so its more obvious as well.
good point with `clipsToBounds`, I was not aware of this flag.

Reply:
funny enough:
```
            if !view.clipsToBounds {
                return
            }
```

`clipsToBounds` is false for the window.

Reply:
i will leave it without for now